README.adoc

```diff
@@ -60,36 +117,45 @@ Be sure to replace `$GITHUB_USERNAME` with your GitHub user id, and `$GITHUB_ACC
 +
 IMPORTANT: The `id` used in your `settings.xml` *must* be `github-sirius-emfjson` and `github-flow` to match what is used in the POMs.
 
-=== Build steps
+=== Build the frontend & backend components and Sirius Web
 
-1. Clone the Sirius Components repository:
-+
-[source,sh]
-----
-git clone https://github.com/eclipse-sirius/sirius-components.git
-----
-
-2. Build the frontend packages
+1. Clone this repository
+2. Build the frontend packages:
 +
 ----
 npm ci
 npx turbo run build
 ----
 +
 
-NOTE: In order to run tests, use `npx turbo run coverage` and if you want to publish the packages on yalc use `npx turbo run publish:local`. If you want to run the whole Sirius Web frontend, use `npm start`.
+NOTE: In order to run tests, use `npx turbo run coverage`
+
+3. Install the frontend artifacts as static resource to be served by the backend.
+From the root directory of the repository:
++
+[source,sh]
+----
+mkdir -p packages/sirius-web/backend/sirius-web-frontend/src/main/resources/static
+cp -R packages/sirius-web/frontend/sirius-web/dist/* packages/sirius-web/backend/sirius-web-frontend/src/main/resources/static
+----
 
-3. Build the backend components.
-From the `sirius-components/packages` directory:
+4. Build the backend components.
 +
 [source,sh]
 ----
-mvn clean install
+mvn clean install -f packages/pom.xml
 ----
 +
 TIP: If you are behind a proxy, you may get Maven errors about checkstyle.org not being available.
 In this case you need to explicitly disable CheckStyle from the build: `mvn clean install -f releng/org.eclipse.sirius.emfjson.releng/pom.xml -P\!checkstyle`
 
+5. You can find in the output artifacts in the various `target` folders of the backend components and the `dist` folders of the frontend components.
+You could publish those to your maven or npm repository to consume them in other applications.
+We are already publishing those components in the NPM and maven repositories of our Github organization.
++
+Using these instructions, we Spring Boot "fat JAR" in `backend/sirius-web-sample-application/target/sirius-web-sample-application-0.0.1-SNAPSHOT.jar`.
```

Comment:
"we [can find] Spring Boot" I suppose, or something like this.
I know the actual version will always be diffferent, but "0.0.1-SNAPSHOT" feels old/very early version.

Reply:
I've updated this part 👍